src/javascripts/views/loginView.js

```diff
@@ -0,0 +1,59 @@
+import { MESSAGE,ERROR_MESSAGE } from "../constants/constant";
+
+class LoginView{
+    
+    constructor(){
+        this.loginForm = document.querySelector(".login-form");
+        this.buttonElement = document.querySelector(".login-form-button");
+    }
+
+    //----- EVENT LISTENER -----//
+    addEventLogin = (isRequired, isValidForm, isValidAccount) => {
+        this.buttonElement?.addEventListener("click", async(e)=>{
+            e.preventDefault();
+
+            const accountInput = {
+                email: this.loginForm.email.value,
+                password: this.loginForm.password.value,
+            }
+
+            const emailInput = document.getElementById("email");
```

Comment:
Pleas try to get elements by its container

Reply:
@phuocduong-agilityio The reason I use it here is because I want to show the error messages at the bottom of the email and password inputs like this ( image ). If I use their container such as class "login-form-input" , then I can only show error messages for email input because email and password both use the same class.
![image](https://github.com/HoangLiarPhuongLe/javascript__training/assets/110210274/3c79c2fb-1088-4401-aa10-f5f0c14323ed)

Reply:
There are many way to do this.
1. Set specific class name for error element
2. Set wrapper that contain input and error element
For your code, we do not know what p element for, because it's not include class name or anything describe for it.